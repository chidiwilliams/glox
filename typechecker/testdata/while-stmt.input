var x = 5;
while (x >= "hello") {
	x = x - 1;
}
