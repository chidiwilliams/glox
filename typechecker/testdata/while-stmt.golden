expected 'number' type for {hello} in {{25 x %!s(<nil>)} 22 >= %!s(<nil>) {hello}}, but got 'string'
